packages/react-native/.gitignore

```diff
@@ -31,3 +31,4 @@ DerivedData
 *.ipa
 *.xcuserstate
 project.xcworkspace
+android
```

Comment:
This is a bit too broad - it currently ignores _all_ files in the `android` directory, whereas it's only necessary to ignore the directory containing the artefacts.

Reply:
Changed to android/com/bugsnag